GenderPayGap.WebUI/Controllers/ScopeNewController.cs

```diff
@@ -132,6 +130,48 @@ public IActionResult ConfirmOutOfScopeAnswers(string encryptedOrganisationId, in
 
             return RedirectToAction("ManageOrganisation", "Organisation", new { id = encryptedOrganisationId });
         }
+        
+        [HttpPost("{encryptedOrganisationId}/reporting-year/{reportingYear}/change-scope/in/confirm")]
+        public IActionResult ConfirmInScopeAnswers(string encryptedOrganisationId, int reportingYear, OutOfScopeViewModel viewModel)
+        {
+            long organisationId = DecryptOrganisationId(encryptedOrganisationId);
+
+            // Get Organisation and OrganisationScope for reporting year
+            Organisation organisation = dataRepository.Get<Organisation>(organisationId);
+
+            User currentUser = ControllerHelper.GetGpgUserFromAspNetUser(User, dataRepository);
+            DateTime currentSnapshotDate = organisation.SectorType.GetAccountingStartDate();
+            
+            organisation.OrganisationScopes.Where(o => o.SnapshotDate.Year == reportingYear).ForEach(s => s.Status = ScopeRowStatuses.Retired);
+            
+            organisation.OrganisationScopes.Add(
+                new OrganisationScope {
+                    OrganisationId = organisation.OrganisationId,
+                    ScopeStatus = ScopeStatuses.InScope,
+                    StatusDetails = "Generated by the system",
+                    ContactFirstname = currentUser.Firstname,
+                    ContactLastname = currentUser.Lastname,
+                    ContactEmailAddress = currentUser.EmailAddress,
+                    Status = ScopeRowStatuses.Active,
+                    SnapshotDate = organisation.SectorType.GetAccountingStartDate(reportingYear)
+                });
+
+            dataRepository.SaveChangesAsync().Wait();
+            
+            // Send emails if scope changed on current reporting year
+            if (viewModel.ReportingYear == currentSnapshotDate)
+            {
+                IEnumerable<string> emailAddressesForOrganisation = organisation.UserOrganisations
+                    .Where(uo => uo.PINConfirmedDate.HasValue)
+                    .Select(uo => uo.User.EmailAddress);
+                foreach (string emailAddress in emailAddressesForOrganisation)
+                {
+                    emailSendingService.SendScopeChangeOutEmail(emailAddress, organisation.OrganisationName);
+                }
+            }

```

Comment:
This code is very similar to the Out of Scope journey - it would be good to commonise it.
You could pull the whole lot into a method, or alternatively, pull each component into a method

e.g.
* Retire old scopes for this reporting year
* Add new scope
* Get email addresses for organisation

Have a think about what will be easiest to read/understand.
We're focussing on clarify, not code size, so it would be good to avoid lots of...
```
if (in scope) {
    do X
} else {
    do Y
}
```
...where possible